config/routes.rb

```diff
@@ -1,19 +1,14 @@
 Rails.application.routes.draw do
-  # get 'users/:user_id/posts' => 'posts#index'
-  # get 'users/:user_id/posts/:id' => 'posts#show'
   root 'users#index'
-  # get 'users/:id' => 'users#show'
   # more readable and clean
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create, :destroy]
```

Comment:
- [ ] [REQUIRED] Kindly add the route for the `:new` action as stated by the project requirement so that we can now have 
``` ruby
resources :comments, only: [:create, :new, :destroy]
```